homeassistant/components/zwave_js/strings.json

```diff
@@ -385,6 +385,44 @@
           "description": "The Notification Event number as defined in the Z-Wave specs."
         }
       }
+    },
+    "set_lock_configuration": {
+      "name": "Set lock configuration",
+      "description": "Sets the configuration for a lock.",
+      "fields": {
+        "operation_type": {
+          "name": "Operation Type",
+          "description": "The operation type of the lock."
+        },
+        "lock_timeout": {
+          "name": "Lock timeout",
+          "description": "Seconds until lock mode times out. Should only be used if operation type is `timed`."
+        },
+        "outside_handles_can_open_door_configuration": {
+          "name": "Outside handles can open door configuration",
+          "description": "A list of four booleans which indicate which outside handles can open the door."
```

Comment:
Do we know more details what the four booleans mean? Why are there four and what does the order mean?

Reply:
Is it clear to the user which handle is 1-4 on these devices? Why are there even 4 handles? I'd expect one handle on each side of the door.

Reply:
You need a better door Martin 😜

Reply:
> Why are there even 4 handles?

Good question. I don't know.

Reply:
should we add something like "Not all locks support this parameter, but if yours does, refer to its user manual for more information." ?

Reply:
Can we find an example of a manual of a device with this feature and see how they describe it?

Reply:
It doesn't look like anyone has an example to work off of. I have reached out to Yale support on the off chance that they have some information but my confidence is low that we will find something useful

Reply:
I'm reading the certification issue again, and a comment was that we probably don't need to expose this feature to the user. If so, I suggest we remove it for now since we don't really know what it does.